blocks/url-input/button.js

```diff
@@ -40,7 +40,7 @@ class UrlInputButton extends Component {
 		const { expanded } = this.state;
 
 		return (
-			<li className="blocks-url-input__button">
+			<div className="blocks-url-input__button">
```

Comment:
I double checked and it looks like it is only used in the toolbar at the moment. It makes a lot of sense to use `div` here 👍